pom.xml

```diff
@@ -31,19 +31,6 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
-		<dependency>

```

Comment:
defining the data source
https://www.baeldung.com/spring-boot-failed-to-configure-data-source